matchmake-extension/database/get_playing_session.go

```diff
@@ -0,0 +1,93 @@
+package database
+
+import (
+	"database/sql"
+
+	"github.com/PretendoNetwork/nex-go/v2"
+	"github.com/PretendoNetwork/nex-go/v2/types"
+	common_globals "github.com/PretendoNetwork/nex-protocols-common-go/v2/globals"
+	match_making_types "github.com/PretendoNetwork/nex-protocols-go/v2/match-making/types"
+)
+
+// GetPlayingSession returns the playing sessions of the given PIDs
+func GetPlayingSession(manager *common_globals.MatchmakingManager, listPID []types.PID) ([]match_making_types.PlayingSession, *nex.Error) {
+	playingSessions := make([]match_making_types.PlayingSession, 0)
```

Comment:
In my testing only up to [`MAX_MATCHMAKE_SESSION_BY_PARTICIPANT`](https://github.com/tech-ticks/RTDXTools/blob/232a7797e01369e9c12704f58fdde65dd3ac1c32/Assets/Scripts/Stubs/Generated/Assembly-CSharp/NexPlugin/Matchmake.cs#L32) is returned